drivers/QSPI.cpp

```diff
@@ -268,11 +269,11 @@ void QSPI::_build_qspi_command(int instruction, int address, int alt)
     //Set up alt phase parameters
     _qspi_command.alt.bus_width = _alt_width;
     _qspi_command.alt.size = _alt_size;
-    if(alt != -1) {
+    if (alt != -1) {
         _qspi_command.alt.value = alt; 
     } else {
         //In the case alt phase is absent, set the alt size to be NONE
-        _qspi_command.alt.value = 0;
+        _qspi_command.alt.size = QSPI_CFG_ALT_SIZE_NONE;
```

Comment:
As we can see, this is error prone, if alt != -1 (its valid) set its value, otherwise size to NONE. Same was for address, and now instruction above does not follow the same style (if instruction phase should be skipped we use instruction.value = 0). Please consider my patch for addressing this issues as we need to have this clear in driver and HAL - how to skip a format phase (instruction, address, alt, data).